pytest_tests/resources/load_params.py

```diff
@@ -1,9 +1,9 @@
 import os
 
 # Load node parameters
-LOAD_NODES = os.getenv("LOAD_NODES", "").split(",")
-LOAD_NODE_SSH_USER = os.getenv("LOAD_NODE_SSH_USER", "root")
-LOAD_NODE_SSH_PRIVATE_KEY_PATH = os.getenv("LOAD_NODE_SSH_PRIVATE_KEY_PATH")
+# LOAD_NODES = os.getenv("LOAD_NODES", "").split(",")
+LOAD_NODE_SSH_USER = os.getenv("LOAD_NODE_SSH_USER", "service")
+LOAD_NODE_SSH_PRIVATE_KEY_PATH = os.getenv("LOAD_NODE_SSH_PRIVATE_KEY_PATH", "./pytest_tests/resources/load_ssh_key")
```

Comment:
The key will work without specifying if it placed to correct path and will be picked up by paramiko automatically.